src/styles/box.scss

```diff
@@ -0,0 +1,22 @@
+
+.box {
+    display: block;
+    background: var(--background-content);
```

Comment:
you have to use our existing SCSS variables in this project, NOT just copy over CSS custom properties from other projects